models/Document/Editable.php

```diff
@@ -557,7 +557,7 @@ public static function buildEditableName(string $type, string $name, Document $d
         // targeting prefix if configured on the document. hasBlocks() determines if
         // there are any parent blocks for the current element
         $targetGroupEditableName = null;
-        if ($document && class_exists(TargetingDocumentInterface::class) && $document instanceof TargetingDocumentInterface) {
+        if ($document && interface_exists(TargetingDocumentInterface::class) && $document instanceof TargetingDocumentInterface) {
```

Comment:
Do we actually need this check? Shouldn't this be enough?

```suggestion
        if ($document instanceof TargetingDocumentInterface) {
```